src/python/bot/fuzzers/engine.py

```diff
@@ -172,3 +177,16 @@ def get(name):
     return engine_class()
 
   return None
+
+
+def do_trial():
+  """Returns or not we should trial using this new implementation."""
+  if environment.get_value('USE_NEW_ENGINE_IMPL'):
```

Comment:
Shouldn't USE_NEW_ENGINE_IMPL be used together with trial ? Is there a place you are enabling 100% ? Can also try on clusterfuzz-aarya-test project?

Reply:
Yes, I will be enabling this for a few jobs manually to better test things.

Reply:
(To better explain my comment).

For all jobs, we will have a 20% chance of fuzzing. For certain jobs, I want to enable this completely, including reproduction etc

Reply:
SGTM.